### ISSUE TITLE
`wg-infra` Status Update 2020-02-03

### ISSUE THREAD

**Issue by rsimha** [ampproject/wg-infra#25]
### What we did
- Updated amp.dev [release schedule](https://amp.dev/documentation/guides-and-tutorials/learn/spec/release-schedule/) page
- Release tagging is live (`PR Use: In Stable / Beta / Experimental`)
- Updated cherry-pick template with new release channel names
- Added a way to build only the core runtime via `gulp build | dist | watch --core_runtime_only`
- Clarified process for adding `amphtml` dependencies using `yarn`
- Triaged backlog of infrastructure issues
- Renaming `AMP_CANARY` cookie to `__Host-AMP_OPT_IN`

### What's next
- Default minified build (`gulp dist`) will now add `AMP_CONFIG` to runtime files
- Performance tests as part of CI / release workflow
- Make LTS and nightly channel widely available
- Take steps to reduce CI flakiness (sauce labs, percy)

**Comment by rcebulko** [comment]
Also release automation for most of the AMP GitHub apps

**Comment by rcebulko** [comment]
Also lots of copy and editing work on the AMP Release blog post by @nainar and @rsimha

**Comment by alanorozco** [comment]
Is `--core_runtime_only` different than doing `--extensions=amp-img`?

**Comment by rsimha** [comment]
> Is `--core_runtime_only` different than doing `--extensions=amp-img`?

Yes, it only builds `amp.js`.
 
<img width="985" alt="Screen Shot 2020-02-03 at 10 54 44 AM" src="https://user-images.githubusercontent.com/26553114/73681645-a6b0a800-4673-11ea-86b6-31976e2885b4.png">

**Comment by dreamofabear** [comment]
Any high-level plans for performance tests that can be shared yet? E.g. which metrics and how to manage data noise. 

@rcebulko Curious what release automation for our GH apps is. A bit of an overloaded term. 😄

**Comment by rsimha** [comment]
> Any high-level plans for performance tests that can be shared yet? E.g. which metrics and how to manage data noise.

@estherkim is meeting with folks from @ampproject/wg-performance this week. We should have more on this in the next update.

**Comment by rcebulko** [comment]
@choumx See https://github.com/ampproject/amp-github-apps/issues/664

TL;DR - Previously only the author of each app (owners, bundle-size, test-status, pr-deploy, etc) could deploy an update in most cases, because it relied on manually deploying to GCloud with a `.env` with secrets and configs that couldn't be checked in and were rarely shared efficiently. Now any of us can push a tag and deploy an app using encrypted secrets and google cloud build.
